components/NavBar/NavBar.jsx

```diff
@@ -1,52 +1,81 @@
 
 const NavBar = (props) => {
+  const classes = useStyles();
   return (
-    <header className={styles.navbar}>
-      <nav className={styles.navbar_navigation}>
-        <div className={styles.navbar_toggle_button}>
-          <DrawerToggleButton click={props.drawerClickHandler} />
-        </div>
-        <div className={styles.navbar_logo}>
-          <img
-            src="/images/SocietyLogo.png"
-            alt="society logo"
-            className={styles.image}
-          />
-        </div>
-        <div className={styles.spacer}></div>
-        <div className={styles.navbar_items}>
-          <ul id="nav-mobile" className="center">
-            <li>
-              <Link href="/">Home</Link>
-            </li>
-            <li>
-              <Link href="/events">Events</Link>
-            </li>
+    <ThemeProvider theme={theme}>
+      <header className={styles.navbar}>
+        <nav className={styles.navbar_navigation}>
+          <div className={styles.navbar_toggle_button}>
+            <DrawerToggleButton click={props.drawerClickHandler} />
+          </div>
+          <div className={styles.navbar_logo}>
+            <img
+              src="/images/SocietyLogo.png"
+              alt="society logo"
+              className={styles.image}
+            />
+          </div>
+          <div className={styles.spacer}></div>
+          <div className={styles.navbar_items}>
+            <ul id="nav-mobile" className="center">
+              <li>
+                <Link href="/">Home</Link>
+              </li>
+              <li>
+                <Link href="/events">Events</Link>
+              </li>
 
-            <div className={styles.dropdown}>
-              <Link className={styles.navbar_items} href="/#ourTeam">
-                Our Team
-              </Link>
-              <div className={styles.dropdown_content}>
-                <a href="/department/operations">Operations</a>
-                <a href="/department/machinelearning">Machine Learning</a>
-                <a href="/department/blockchain">Blockchain</a>
-                <a href="/department/devops">DevOps</a>
+              <div className={styles.dropdown}>
+                <Link className={styles.navbar_items} href="/#ourTeam">
+                  Our Team
+                </Link>
+                <div className={styles.dropdown_content}>
+                  <a href="/department/operations">Operations</a>
+                  <a href="/department/machinelearning">Machine Learning</a>
+                  <a href="/department/blockchain">Blockchain</a>
+                  <a href="/department/devops">DevOps</a>
+                </div>
               </div>
-            </div>
 
-            <li>
-              <Link href="/#contact">Contact Us</Link>
-            </li>
-          </ul>
+              <li>
+                <Link href="/#contact">Contact Us</Link>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <div>
+          <Alert
```

Comment:
I was thinking instead of placing the Alert Component here directly, we create a separate component called Notice with the implementation, and import it here when it is being used. This would be neater since the Notice component will not be shown just yet.

Reply:
Yes, thank you for the feedback